server/monitor.go

```diff
@@ -2768,6 +2768,7 @@ func (s *Server) HandleHealthz(w http.ResponseWriter, r *http.Request) {
 
 	hs := s.healthz()
 	if hs.Error != _EMPTY_ {
+		s.Errorf("Healthcheck failed: %q", hs.Error)
```

Comment:
Might suggest this be a Warnf since it can and should clear.

Reply:
:+1: